File: evenements/src/eventBundle/Entity/pack.php
<?php

namespace eventBundle\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * pack
 *
 * @ORM\Table(name="pack")
 * @ORM\Entity(repositoryClass="eventBundle\Repository\packRepository")
 */
class pack
{
    /**
     * @var int
     *
     * @ORM\Column(name="id", type="integer")
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="AUTO")
     */
    private $id;

    /**
     * @var string
     *
     * @ORM\Column(name="titre", type="string", length=255)
     */
    private $titre;

    /**
     * @var string
     *
     * @ORM\Column(name="description", type="string", length=255)
     */
    private $description;

    /**
     * @var string
     *
     * @ORM\Column(name="type", type="string", length=255)
     */
    private $type;

    /**
     * @var int
     *
     * @ORM\Column(name="reduction", type="integer")
     */
    private $reduction;

    /**
     * @var string
     *
     * @ORM\Column(name="image", type="string", length=255)
     */
    private $image;

    /**
     * @var \DateTime
     *
     * @ORM\Column(name="duree_du", type="datetime")
     */
    private $dureeDu;

    /**
     * @var \DateTime
     *
     * @ORM\Column(name="duree_a", type="datetime")
     */
    private $dureeA;


    /**
     * Get id
     *
     * @return int
     */
    public function getId()
    {
        return $this->id;
    }

    /**
     * Set titre
     *
     * @param string $titre
     *
     * @return pack
     */
    public function setTitre($titre)
    {
        $this->titre = $titre;

        return $this;
    }

    /**
     * Get titre
     *
     * @return string
     */
    public function getTitre()
    {
        return $this->titre;
    }

    /**
     * Set description
     *
     * @param string $description
     *
     * @return pack
     */
    public function setDescription($description)
    {
        $this->description = $description;

        return $this;
    }

    /**
     * Get description
     *
     * @return string
     */
    public function getDescription()
    {
        return $this->description;
    }

    /**
     * Set type
     *
     * @param string $type
     *
     * @return pack
     */
    public function setType($type)
    {
        $this->type = $type;

        return $this;
    }

    /**
     * Get type
     *
     * @return string
     */
    public function getType()
    {
        return $this->type;
    }

    /**
     * Set reduction
     *
     * @param integer $reduction
     *
     * @return pack
     */
    public function setReduction($reduction)
    {
        $this->reduction = $reduction;

        return $this;
    }

    /**
     * Get reduction
     *
     * @return int
     */
    public function getReduction()
    {
        return $this->reduction;
    }

    /**
     * Set image
     *
     * @param string $image
     *
     * @return pack
     */
    public function setImage($image)
    {
        $this->image = $image;

        return $this;
    }

    /**
     * Get image
     *
     * @return string
     */
    public function getImage()
    {
        return $this->image;
    }

    /**
     * Set dureeDu
     *
     * @param \DateTime $dureeDu
     *
     * @return pack
     */
    public function setDureeDu($dureeDu)
    {
        $this->dureeDu = $dureeDu;

        return $this;
    }

    /**
     * Get dureeDu
     *
     * @return \DateTime
     */
    public function getDureeDu()
    {
        return $this->dureeDu;
    }

    /**
     * Set dureeA
     *
     * @param \DateTime $dureeA
     *
     * @return pack
     */
    public function setDureeA($dureeA)
    {
        $this->dureeA = $dureeA;

        return $this;
    }

    /**
     * Get dureeA
     *
     * @return \DateTime
     */
    public function getDureeA()
    {
        return $this->dureeA;
    }
}

File: CleanModernUIKit-master/src/com/mycompany/Entite/Pack.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.mycompany.Entite;

/**
 *
 * @author Ammouna_Zikou
 */
public class Pack {
    private double id , reduction ;
    private String titre , description , type , image , dureeDu , dureeA;

    public Pack(int reduction, String titre, String description, String type, String dureeDu, String dureeA) {
        this.reduction = reduction;
        this.titre = titre;
        this.description = description;
        this.type = type;
        this.dureeDu = dureeDu;
        this.dureeA = dureeA;
    }

    public Pack() {
    }

    public Pack(int id, int reduction, String titre, String description, String type, String dureeDu, String dureeA) {
        this.id = id;
        this.reduction = reduction;
        this.titre = titre;
        this.description = description;
        this.type = type;
        this.dureeDu = dureeDu;
        this.dureeA = dureeA;
    }

    public double getId() {
        return id;
    }

    public void setId(double id) {
        this.id = id;
    }

    public double getReduction() {
        return reduction;
    }

    public void setReduction(double reduction) {
        this.reduction = reduction;
    }

    public String getTitre() {
        return titre;
    }

    public void setTitre(String titre) {
        this.titre = titre;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }

    public String getImage() {
        return image;
    }

    public void setImage(String image) {
        this.image = image;
    }

    public String getDureeDu() {
        return dureeDu;
    }

    public void setDureeDu(String dureeDu) {
        this.dureeDu = dureeDu;
    }

    public String getDureeA() {
        return dureeA;
    }

    public void setDureeA(String dureeA) {
        this.dureeA = dureeA;
    }

    @Override
    public String toString() {
        return "Pack{" + "id=" + id + ", reduction=" + reduction + ", titre=" + titre + ", description=" + description + ", type=" + type + ", image=" + image + ", dureeDu=" + dureeDu + ", dureeA=" + dureeA + '}';
    }
    
    
    
}
